### ISSUE TITLE
Make Spacebar pause the video

### ISSUE THREAD

**Issue by alex-fu27** [joschahenningsen/TUM-Live#521]
**Is your feature request related to a problem? Please describe.**
I always forget that the spacebar key is not mapped to anything, so I accidentally push it when I want to pause the video. Then often I get confused by what the video player does. For example, the most annoying thing that can happen is if the last button I clicked was the "skip pause" button, then pushing spacebar will send me back to the end of the last pause.

**Describe the solution you'd like**
It would be great if spacebar toggled pause/play as I am used to because of Youtube/VLC/etc.

**Comment by just-max** [comment]
I actually added keyboard shortcuts in #476, so the space bar usually does play/pause the video (if shortcuts aren't working for you in general, that's a different issue of course). However, if a button has focus, that button will be activated instead (this makes it possible to control the player with only the keyboard by using tab+space bar). YouTube even [claims](https://support.google.com/youtube/answer/7631406?hl=en) to have the same behaviour, but pressing space to activate a button seems to *only* work when a button has been selected with the tab key (otherwise the space bar always plays/pauses as you are expecting).

The specific case of the skip pause button seems like a bug/missed edge case though.

So TL;DR: you can use `k` to play/pause instead or make sure no button is selected.

**Comment by joschahenningsen** [comment]
Removing the current behavior (having a button clicked on space-press when it has focus) would impact accessibility. Visually impaired users might rely on this for navigation.

I will however make sure that the 'skip pause' button is destroyed or unclickable when not shown. Thanks for the hint on that!
